tests/core/create-user-2/task.yaml

```diff
@@ -1,7 +1,6 @@
 summary: Ensure create-user functionality
 
-# TODO:UC20: enable on UC20
-systems: [ubuntu-core-1*]
+systems: [ubuntu-core-*]
```

Comment:
this line is now redundant, only ubuntu-core-* systems are available for the core suite

```suggestion
```

Reply:
done